gradle.properties

```diff
@@ -2,17 +2,17 @@
 org.gradle.jvmargs=-Xmx1G
 # Fabric Properties
 # check these on https://fabricmc.net/develop/
-minecraft_version=1.19.4
-yarn_mappings=1.19.4+build.1
-loader_version=0.14.17
+minecraft_version=1.20.1
+yarn_mappings=1.20.1+build.9
+loader_version=0.14.21
 # Mod Properties
-mod_version=1.12+mc1.19.4
+mod_version=1.12+mc1.20.1
 maven_group=me.juancarloscp52
 archives_base_name=Entropy
 # Dependencies
-fabric_version=0.75.3+1.19.4
+fabric_version=0.85.0+1.20.1
```

Comment:
This definitely seems worthwhile to have at latest
`fabric_version=0.92.3+1.20.1`

Reply:
Updating to that version requires a newer Loom version, so I'll go ahead and update all of that, too. Since this PR already contains build file changes, it's probably fine to do.